src/basicElements/serializers/number/integers.ts

```diff
@@ -67,3 +78,44 @@ export function numberToUnsigned(
   mustBeValidUnsigned(sizeInBits, int)
   return int
 }
+
+function setBigUint128(
+  view: DataView,
+  byteOffset: number,
+  value: bigint
+): void {
+  view.setBigUint64(byteOffset, value & U64.MAX, true)
+  view.setBigUint64(
+    byteOffset + U8.SIZE_BIT,
+    value >> BigInt(U64.SIZE_BIT),
+    true
+  )
+}
+
+function getBigUint128(view: DataView, byteOffset: number): bigint {
+  const low = view.getBigUint64(byteOffset, true)
+  byteOffset += U8.SIZE_BIT
+  const high = view.getBigUint64(byteOffset, true)
+  return (high << BigInt(U64.SIZE_BIT)) | low
+}
+
+function setBigUint256(
+  view: DataView,
+  byteOffset: number,
+  value: bigint
+): void {
+  for (let i = ZERO; i < U256_NUM_64_BIT_CHUNKS; i++) {
+    view.setBigUint64(byteOffset, value & U64.MAX, true)
+    byteOffset += U64.SIZE_BYTE
+    value >>= BigInt(U64.SIZE_BIT)
+  }
+}
+
+function getBigUint256(view: DataView, byteOffset: number): bigint {
+  let result = BigInt(ZERO)
+  for (let i = U256_NUM_64_BIT_CHUNKS - ONE; i >= ZERO; i--) {
+    const part = view.getBigUint64(byteOffset + i * U64.SIZE_BYTE, true)
+    result = (result << BigInt(U64.SIZE_BIT)) | part
+  }
```

Comment:
Reverse loop make the code more complex to read IMHO.
Did you consider:
```suggestion
  for (let i = ZERO; i < U256_NUM_64_BIT_CHUNKS; i++) {
    const part = view.getBigUint64(byteOffset + i * U64.SIZE_BYTE, true)
    result |= part << BigInt(U64.SIZE_BIT) * i
  }
```